app/src/main/java/ch/epfl/qedit/model/QuizCategorie.java

```diff
@@ -0,0 +1,6 @@
+package ch.epfl.qedit.model;
+
+public enum QuizCategorie {
```

Comment:
Maybe, with a list of questions, we don't really need this enum

Reply:
Yep